### DISCUSSION TITLE
HyperHDR Script?

### DISCUSSION THREAD

**Discussion by DJFlaxi** [tteck/Proxmox#2358] [Q&A] [answered]
Hello,

can you make an Script for HyperHDR?
(HyperHDR.eu or https://github.com/awawa-dev/HyperHDR/releases)

I want to install it on Proxmox but i dont know how. 

Thx
Marc

**Comment by tteck** [comment, marked as answer]
Ready for testing
```
bash -c "$(wget -qO - https://github.com/tteck/Proxmox/raw/main/ct/hyperhdr.sh)"
```

**Comment by tteck** [comment]
You're welcome.

**Comment by DJFlaxi** [comment]
Hi tteck,

wow you are very fast....

The script doesn't allow the default mode. Only Advanced Mode works.
Can you tell me how I bind in the USB-Stick (Grapper)?

Thx
